modules/programs/msmtp.nix

```diff
@@ -26,9 +26,8 @@ let
         tls_fingerprint = msmtp.tls.fingerprint;
       } // optionalAttrs (smtp.port != null) { port = toString smtp.port; }
         // optionalAttrs (passwordCommand != null) {
-          # msmtp requires the password to finish with a newline.
           passwordeval =
-            ''${pkgs.bash}/bin/bash -c "${toString passwordCommand}; echo"'';
+            ''${pkgs.bash}/bin/bash -c "${toString passwordCommand}"'';
```

Comment:
Isn't this the same as

    passwordeval = toString passwordCommand

?

Reply:
Yes, you're right. Overlooked that :facepalm: Since this is already merged, created a new PR: https://github.com/nix-community/home-manager/pull/1649